lib/bolt/puppetdb/config.rb

```diff
@@ -6,7 +6,8 @@ module Bolt
   module PuppetDB
     class Config
       DEFAULT_TOKEN = File.expand_path('~/.puppetlabs/token')
-      DEFAULT_CONFIG = File.expand_path('~/.puppetlabs/client-tools/puppetdb.conf')
+      DEFAULT_CONFIG = { user: File.expand_path('~/.puppetlabs/client-tools/puppetdb.conf'),
+                         global: '/etc/puppetlabs/client-tools/puppetdb.conf' }.freeze
```

Comment:
This location different between Windows and *nix. See [puppet-specifications](https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md#puppet-agent-windows) for more detail.

Reply:
Here's an example of how we check for Windows now:

https://github.com/puppetlabs/bolt/blob/15995af05ae4ac189d970b34a783ffeffb3b4927/lib/bolt/transport/ssh/connection.rb#L62

Reply:
Can we add that to Bolt::Util